### PR TITLE
Readded OT splitting after KOS refactor

### PR DESCRIPTION
**Pull request by th4s**
This PR implements OT splitting

### CODE REVIEW

**Comment by themighty1** [comment]
I'll wait for @sinui0 to ACK first, so I could do a security review on the final version.

File: mpc-core/src/ot/extension/kos15/receiver/mod.rs
Comment (th4s) [reply mpc-core/src/ot/extension/kos15/receiver/mod.rs]: yup

**Review by themighty1 — APPROVED** [review]
ACK

**Pull request merged**
